sonar-db/src/main/java/org/sonar/db/permission/PermissionDao.java

```diff
@@ -102,34 +100,32 @@ public int countGroups(DbSession session, String permission, @Nullable Long comp
   /**
    * Each row returns a CountByProjectAndPermissionDto
    */
-  public void usersCountByComponentIdAndPermission(final DbSession dbSession, List<Long> componentIds, final ResultHandler resultHandler) {
-    final Map<String, Object> parameters = new HashMap<>();
+  public void usersCountByComponentIdAndPermission(DbSession dbSession, List<Long> componentIds, ResultHandler resultHandler) {
+    Map<String, Object> parameters = new HashMap<>(1);
 
-    DatabaseUtils.executeLargeInputsWithoutOutput(componentIds, new Function<List<Long>, Void>() {
-      @Override
-      public Void apply(@Nonnull List<Long> partitionedComponentIds) {
+    executeLargeInputsWithoutOutput(
+      componentIds,
+      partitionedComponentIds -> {
         parameters.put("componentIds", partitionedComponentIds);
         mapper(dbSession).usersCountByProjectIdAndPermission(parameters, resultHandler);
         return null;
-      }
-    });
+      });
   }
 
   /**
    * Each row returns a CountByProjectAndPermissionDto
    */
-  public void groupsCountByComponentIdAndPermission(final DbSession dbSession, final List<Long> componentIds, final ResultHandler resultHandler) {
-    final Map<String, Object> parameters = new HashMap<>();
+  public void groupsCountByComponentIdAndPermission(DbSession dbSession, List<Long> componentIds, ResultHandler resultHandler) {
+    Map<String, Object> parameters = new HashMap<>(2);
     parameters.put(ANYONE_GROUP_PARAMETER, DefaultGroups.ANYONE);
 
-    DatabaseUtils.executeLargeInputsWithoutOutput(componentIds, new Function<List<Long>, Void>() {
-      @Override
-      public Void apply(@Nonnull List<Long> partitionedComponentIds) {
+    executeLargeInputsWithoutOutput(
+      componentIds,
+      partitionedComponentIds -> {
         parameters.put("componentIds", partitionedComponentIds);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)